Makefile

```diff
@@ -4,6 +4,12 @@
 PROJECT_NAME := provider-jet-aws
 PROJECT_REPO := github.com/crossplane-contrib/$(PROJECT_NAME)
 
+export TERRAFORM_VERSION := 1.0.5
```

Comment:
Could you make the changes in `cluster/images/provider-jet-aws-controller/Dockerfile` as well like [this](https://github.com/crossplane-contrib/provider-jet-template/blob/main/cluster/images/provider-jet-template-controller/Dockerfile) so that this is the only place we have to declare these variables?

Reply:
done